website/docs/r/org.html.markdown

```diff
@@ -47,6 +58,26 @@ The following arguments are supported:
 * `stored_vm_quota` - (Optional) - Maximum number of virtual machines in vApps or vApp templates that can be stored in an undeployed state by a member of this organization. Default is unlimited (0)
 * `can_publish_catalogs` - (Optional) - True if this organization is allowed to share catalogs. Default is `true`.
 * `delay_after_power_on_seconds` - (Optional) - Specifies this organization's default for virtual machine boot delay after power on. Default is `0`.
+* `vapp_lease` - (Optional; *v2.7*) - Defines lease parameters for vApps created in this organization. See [vApp Lease](#vapp-lease) below for details. 
+* `vapp_template_lease` - (Optional; *v2.7*) - Defines lease parameters for vApp templates created in this organization. See [vApp Template Lease](#vapp-template-lease) below for details.
+
+<a id="vapp-lease"></a>
+## vApp Lease
+
+The `vapp_lease` section contains lease parameters for vApps created in the current organization, as defined below:
+
+* `maximum_runtime_lease_in_sec` - (Optional) - How long vApps can run before they are automatically stopped (in seconds). 0 means never expires. Values accepted from 3600+. - Default is 604800 (7 days)
+* `power_off_on_runtime_lease_expiration` - Optional - When true, vApps are powered off when the runtime lease expires. When false or missing, vApps are suspended when the runtime lease expires.
+* `maximum_storage_lease_in_sec` - (Optional) - How long stopped vApps are available before being automatically cleaned up (in seconds). 0 means never expires. Regular values accepted from 3600+. Default is 1209600 (14 days)
+* `delete_on_storage_lease_expiration` - (Optional) - If true, storage for a vApp is deleted when the vApp's lease expires. If false, the storage is flagged for deletion, but not deleted.
```

Comment:
Documentation should not that false is the default. Same for `vapp_template_lease` section

Reply:
Documentation updated, following some changes in the lease sections: the sections are still optional, but all the fields inside are mandatory. The docs were updated to reflect the new requirements.
The change was necessary because the API return different results depending on whether we put in the whole lease block or not. If we don't put the block, we'll get the same defaults that we get from the GUI (e.g. maximum runtime lease = 7 days) but if we put  other values in the block and omit `maximum_runtime_lease`, the default is zero (never expires).
By making all fields mandatory we avoid the ambiguity.